src/pages/ResultsPage.js

```diff
@@ -0,0 +1,58 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import Button from '../components/Button'
+import CropItem from '../components/CropItem'
+import useFetch from '../hooks/useFetch.js'
+import { useParams } from 'react-router-dom'
+
+ResultsPage.propTypes = {
+  onBack: PropTypes.func.isRequired,
+  onDetails: PropTypes.func.isRequired,
+}
+
+export default function ResultsPage({ onBack, onDetails }) {
+  const { searchTerm } = useParams()
+  const { data, isQuerying } = useFetch(searchTerm)
+  if (isQuerying) return <div>loading...</div>
+  return (
+    <div>
+      <BackButton onClick={onBack}>&lt; back</BackButton>
+      <Output>
+        {data.length !== 0 ? (
+          <>
+            {data
+              ?.filter(crop =>
+                crop.attributes.main_image_path.match(/(https)/gi)
+              )
+              .map(({ id, attributes }) => (
+                <CropItem
+                  key={id}
+                  name={attributes.name}
+                  image={attributes.main_image_path}
+                  onClick={() => onDetails(id)}
+                />
+              ))}
+          </>
+        ) : (
+          <Paragraph>
+            No crop found.
+            <br />
+            Please try again and type at least 3 letters.
+          </Paragraph>
+        )}
+      </Output>
+    </div>
```

Comment:
Good work! UseParams is new for me, but it looks quite helpful to me, to make the code cleaner.